packages/remix-server-runtime/crypto.ts

```diff
@@ -1,9 +1,14 @@
-export type SignFunction = (value: string, secret: string) => Promise<string>;
+import type { Promisable } from "type-fest";
+
+export type SignFunction = (
+  value: string,
+  secret: string
+) => Promisable<string>;
 
 export type UnsignFunction = (
   cookie: string,
   secret: string
-) => Promise<string | false>;
+) => Promisable<string | false>;
```

Comment:
Is this a breaking API change since `unsign` is publicly exported?  Won't `unsign` always return a `Promise` today, so `unsign().then()` works - but after this `unsign` just returns a `string | false` (the return type of `cookieSignature.unsign`) so `unsign().then(` would break?

If so, I think we should keep the type API consistent and change the implemtnation:

```
export const unsign: UnsignFunction = (signed: string, secret: string) => {
  return Promise.resolve(cookieSignature.unsign(signed, secret));
};
```

Reply:
@brophdawg11 Indeed a good one I didn't think of

Wrapping it in a `Promise` would indeed be the better option in this case I think @JoshuaKGoldberg

Reply:
Makes sense! I'll change the functions to `Promise.resolve(...)`. Thanks!